test_paths_test.go

```diff
@@ -96,7 +96,8 @@ func TestResultsTestPaths(t *testing.T) {
 
 				for _, link := range metadata.Links {
 					for _, result := range link.Results {
-						if result.TestPath != "" {
+						// TODO: support testing wildcard (*) TestPaths
```

Comment:
Nit: shall we have a tracking issue?

Reply:
https://github.com/web-platform-tests/wpt-metadata/issues/76